radon/tests/test_ipynb.py

```diff
@@ -28,7 +28,7 @@
 @pytest.mark.skipif(not SUPPORTS_IPYNB, reason="nbformat not installed")
 def test_harvestor_yields_ipynb(log_mock):
     '''Test that Harvester will try ipynb files when configured'''
-    target = os.path.join(DIRNAME, 'data/example.ipynb')
+    target = os.path.join(DIRNAME, 'data\\example.ipynb')
```

Comment:
This should not be required on Windows, as forward slashes work fine there, as far as I know. But it does break things on Unix platforms, so it has to be reverted.

Reply:
I was getting a pytest error with the forward slash:

```
log_mock = <MagicMock name='log_result' id='1643343259728'>

    @pytest.mark.skipif(not SUPPORTS_IPYNB, reason="nbformat not installed")
    def test_harvestor_yields_ipynb(log_mock):
        '''Test that Harvester will try ipynb files when configured'''
        target = os.path.join(DIRNAME, 'data/example.ipynb')
        harvester = Harvester([DIRNAME], BASE_CONFIG_WITH_IPYNB)
        filenames = list(harvester._iter_filenames())
        assert _is_python_file(target)
        assert len(filenames) == 1
>       assert target in filenames
E       AssertionError: assert 'C:\\Users\\b_gra\\Desktop\\projects\\2020\\radon\\radon\\tests\\data/example.ipynb' in ['C:\\Users\\b_gra\\Desktop\\projects\\2020\\radon\\radon\\tests\\data\\example.ipynb']
```

Reply:
@BlaneG I see. But if we hardcode the separator like that the tests will fail on linux. You can keep the forward-slash and use [`os.path.normpath`](https://docs.python.org/3/library/os.path.html#os.path.normpath).

Reply:
`os.path.normpath`? I would personally use the `pathlib` API, but that's just me.